### PR TITLE
Added Service Invocation Rest API swagger docs

### PR DESCRIPTION
**Pull request by dancadana**
# Description

Added swagger.json to describe dapr Service Invocation API

## Issue reference

https://github.com/dapr/dapr/issues/16#event-3544782915

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [X ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#16
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: dapr/proto/swagger.json [hunk 1]
```diff
@@ -0,0 +1,389 @@
+{
```
Comment (amanbha) [inline dapr/proto/swagger.json:1]: could you please move it to dapr/swagger?

**Comment by dancadana** [comment]
moved to dapr/swagger folder as requested.

**Comment by yaron2** [comment]
Thanks for this @dancadana, and welcome to the Dapr community.

**Review by youngbupark — COMMENTED** [review]
@dancadana I know this pr is merged, but I found some typos and incorrect parts this pr. please create new pr to address the comments.

File: swagger/swagger.json [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+            "$ref": "#/components/parameters/Content-Type"
+          },
+          {
+            "$ref": "#/components/securitySchemes/ApiKeyAuth",
+            "name": "ApiKeyAuth",
+            "x-last-modified": 1597270199372
+          }
+        ],
+        "responses": {
+          "200": {
```
[the remark this answers is not in the document]
Comment (dancadana) [reply swagger/swagger.json:58]: Documentation mentions those.  https://github.com/dapr/docs/blob/master/reference/api/service_invocation_api.md
So I thought to add those. Not sure how to answer your question. Fixed all the rest problems you found. Please let me know what to do with response codes. Maybe @yaron2  would know.

**Pull request merged**